src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatComponents = props.entries.map((entry) => {
+        return (
+            <ChatEntry
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            updateChatData={props.updateChatData}
+            />
+        );
+    });
+
+    return (
+        <section>
+            {chatComponents}
+        </section>
+    )
+};
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(PropTypes.shape({
+        id: PropTypes.number.isRequired,
+        sender: PropTypes.string.isRequired,
+        body: PropTypes.string.isRequired,
+        timeStamp: PropTypes.string.isRequired,
+        liked: PropTypes.bool.isRequired,
```

Comment:
Notice that the prop `likeCount` that was passed to ChatLog on line 40 in App.js isn't part of PropTypes but also isn't referenced in the rest of this component?